core/src/main/java/org/akaza/openclinica/service/auth/TokenServiceImpl.java

```diff
@@ -55,4 +56,16 @@
             throw new InvalidTokenException("Cannot convert access token to JSON", e);
         }
     }
+
+    public String getRole(String token) {
```

Comment:
this should probably be renamed to getUserType since you are returning user type and not role